ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -521,5 +521,8 @@ open class BadgeView: UIView, TokenizedControlInternal {
 
     // MARK: Accessibility
 
-    open override var accessibilityLabel: String? { get { return label.text } set { } }
+    open override var accessibilityLabel: String? {
+        get { return dataSource?.accessibilityLabel ?? label.text }
```

Comment:
Where does this accessibilityLabel come from in the dataSource? I'm trying to find it in code and I can't track it down.

Reply:
It's the NSObject's `accessibilityLabel`. Originally I tried to add a new variable in dataSource but the compiler didn't like that.